pybaseball/cache/cache_config.py

```diff
@@ -0,0 +1,25 @@
+import pathlib
+from datetime import timedelta
+from os import path
+
+from .cache_type import CacheType
+from .file_utils import _mkdir
+
+class CacheConfig:
```

Comment:
Dumb question: where does this get stored?

Reply:
It's in memory only:

See line 17 of https://github.com/jldbc/pybaseball/pull/128/files#diff-5566a5251165c3b840f204a66a4506a3